src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -1070,7 +1070,18 @@ public function insert_posts( $items = array() ) {
 				$event['post_parent'] = $k;
 			}
 
-			//if we should create a venue or use existing
+			// Do we have an existing venue for this event that we should preserve?
+			// @todo review: should we care about the potential for multiple venue IDs?
```

Comment:
Technically, we support multiple venue ids (if someone unhooks/re-hooks the post type from events). We can leave it as a To-Do for the future in the interest of time. Perhaps ticket the future change and we can prioritize it when there's a usecase or if time allows!